test/_fixtures/error-payload.json

```diff
@@ -0,0 +1,12 @@
+{
+    "fullDisplayName": "node-test-commit-arm #3087",
+    "result": "FAILURE",
+    "changeSet": {
+        "items": [
+            {
+                "commitId": "7190bd23810f0880d95605bf6f785224a345d477"
+            }
+        ]
+    },
```

Comment:
Its really up to us how we want to handle it -- we could test every commit if we wanted to -- but since this will be driven by manual triggers (someone creates a test-pr job) we don't really care what commit it is, just posting an update against it.